specs-go/config.go

```diff
@@ -430,6 +430,8 @@ type SolarisAnet struct {
 
 // Windows defines the runtime configuration for Windows based containers, including Hyper-V containers.
 type Windows struct {
+	// LayerFolders contains a list of absolute paths to directories containing image layers.
+	LayerFolders []string `json:"layerFolders"`
```

Comment:
OK, lets leave this as is. I'll take an action item to clean up the casing once this is merged. Opened https://github.com/opencontainers/runtime-spec/issues/857 to track.